### DISCUSSION TITLE
Yimmenu might be detected

### DISCUSSION THREAD

**Discussion by LintStalker** [YimMenu/YimMenu#2125] [General]
I got a ban email from R* this morning.  I posted in Unknowncheats ban thread, but I thought I would post here as well so that it could be discussed.  a few other Yimmenu users are also posting ban reports.

I was also col loading Kiddions modest menu

I wasn't doing any fast money methods.  Just driving around shooting at cops.

**Comment by lonelybud** [comment]
kiddion moddest menu was also coloaded?

**Reply by Yimura** [reply]
Just running at the same time as the menu I'm assuming.

**Reply by LintStalker** [reply]
Yes, I had Kiddions running at the same time.  

BTW, I closed this discussion and add what I know to yours

**Reply by lonelybud** [reply]
I also use kiddion btw. I recently used `LSC Forge Model`, lua script to unlock skins from gun van, Ultimate_Menu (to unlock Stone Hatchet, casino outfits, instant Ballastic Equipment)

**Comment by lonelybud** [comment]
![image](https://github.com/YimMenu/YimMenu/assets/139460769/11d5f764-2265-49e6-9bf6-e7f9e3dd32e5)

i just saw this msg in game.

**Reply by Yimura** [reply]
Could you leave a comment here: https://github.com/YimMenu/YimMenu/issues/2124

**Reply by lonelybud** [reply]
yes i will do that

**Reply by lonelybud** [reply]
![image](https://github.com/YimMenu/YimMenu/assets/139460769/9244ad08-bebb-4de6-9476-4241020f42ca)

**Comment by iDominiqe** [comment]
![image](https://github.com/YimMenu/YimMenu/assets/115355282/d7bedd92-03df-4f4b-a91e-c0adb6c74ad8)

**Reply by Yimura** [reply]
Please share some information with us on #2124.

**Comment by LintStalker** [comment]
Hi Andreas!

Thank you for closing 2125.  I thought I had, but apparently not.  I'm
rather new to github.

I added my info to the case that Yimura opened.

How is it going with figuring out how it was detected ?   Do you think
there will be a fix soon?

John
